src/main/io/osd.c

```diff
@@ -153,6 +153,13 @@ static char osdGetAltitudeSymbol()
     }
 }
 
+static char osdGetBatterySymbol(int cellVoltage)
+{
+    /* Calculate current battery cell voltage as a percentage of range */
+    int voltPercent = ((cellVoltage - (batteryConfig()->vbatmincellvoltage * 10)) * 7) / ((batteryConfig()->vbatmaxcellvoltage - batteryConfig()->vbatmincellvoltage) * 10);
```

Comment:
What about mapping the range `vbatwarningcellvoltage` to `vbatmaxcellvoltage` over the symbols above empty and show the empty symbol for cell voltage less than `vbatwarningcellvoltage`?

This would ensure the battery is only depicted as "empty" when the voltage is at a not OK level.